allennlp/__main__.py

```diff
@@ -0,0 +1,26 @@
+from typing import Sequence
+import argparse
+import logging
+import sys
+
+from allennlp.commands.bulk import add_bulk_subparser
+from allennlp.common.params import PARAMETER
+
+# disable parameter logging
+logging.disable(PARAMETER)
```

Comment:
that output wasn't from flask

without that line I get all the 

```
text_field_embedder = {'tokens': {'embedding_dim': 5, 'type': 'embedding'}}
text_field_embedder.type = basic
text_field_embedder.tokens = {'embedding_dim': 5, 'type': 'embedding'}
text_field_embedder.tokens.type = embedding
...
```

param stuff in the output of the command-line tool, which I don't want, although I will want that if I train a model, so maybe I should move that into bulk.  I'll do that.